src/components/views/ProductPage/ProductPage.module.scss

```diff
@@ -193,3 +193,49 @@
     }
   }
 }
+
+.menu {
+  background-color: gray;
+  margin-bottom: 50px;
+  height: 400px;
+
+  .furniture {
+    color: $topBar-color-a;
+    font-size: 21px;
+  }
+
+  ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    text-transform: capitalize;
+
+    li {
+      display: inline-block;
+      line-height: 85px;
+      margin: 0 5px 0 0;
+      color: white;
+      font-size: 30px;
+
+      a {
+        @extend %hover-effect;
+        color: $topBar-color-a;
+        font-size: 20px;
+
+        &:hover {
+          color: $topBar-color;
+        }
+      }
+    }
+
+    a::before {
+      content: "\00BB";
+    }
+  }
+}
+
+.product {
```

Comment:
Tutaj należy dodać ` padding: $base-padding 0;` i jednocześnie usunąć padding z  selektora `.root`. Piwerwsza wersja strony produktowej nie zawierała tego top bara i była tylko na białym tle, ale jak doszedł nagłówek, to zgodnie z projektem pomiędzy nagłówkiem a głóewnym menu nie ma przestrzeni

![image](https://user-images.githubusercontent.com/7238770/96359473-89ddd700-1113-11eb-92b6-7bb6d70f06e2.png)